app/rest/db-logging/src/main/java/io/syndesis/rest/dblogging/jaxrs/model/Exchange.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.rest.dblogging.jaxrs.model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.util.List;
+
+/**
+ */
+public class Exchange {
```

Comment:
Exchange sound a bit very generic. Isn't it a 'ExchangeLog' ? (just to differentiate it from a Camel exchange maybe ...)

Reply:
I don't think we have any danger of both this class and a camel Exchange class being both used from the same java class :)

Reply:
Still I don't think it  _is) an 'Exchange' but a log about an exchange, so 'ExchangeLog' would fit much better IMO.

Reply:
But feel free to reject this comment like the others, too.

Reply:
How about ExchangeActivity??